src/util/mapbox.js

```diff
@@ -198,12 +229,12 @@ class TelemetryEvent {
 
     saveEventData() {
         const isLocalStorageAvailable = storageAvailable('localStorage');
-        const storageKey = `${telemEventKey}:${config.ACCESS_TOKEN || ''}`;
-        const uuidKey = `${telemEventKey}.uuid:${config.ACCESS_TOKEN || ''}`;
+        const storageKey =  this.getStorageKey();
+        const uuidKey = this.getStorageKey('uuid');
         if (isLocalStorageAvailable) {
             try {
                 window.localStorage.setItem(uuidKey, this.anonId);
-                if (this.eventData.lastSuccess) {
+                if (Object.keys(this.eventData).length >= 1) {
```

Comment:
Why is this change here? Isn't `lastSuccess` still set in the same place?

Reply:
It is unrelated, but it ensures that state is saved even if it doesn't include `lastSuccess`